test/vault_cluster_enterprise_test.go

```diff
@@ -0,0 +1,16 @@
+package test
+
+import (
+	"os"
+	"testing"
+)
+
+func TestVaultClusterEnterpriseWithUbuntuAmi(t *testing.T) {
+	t.Parallel()
+	runVaultEnterpriseClusterTest(t, "ubuntu16-ami", "ubuntu", os.Getenv("VAULT_AMI_TEMPLATE_VAR_DOWNLOAD_URL"))
+}
+
+func TestVaultClusterEnterpriseWithAmazonLinuxAmi(t *testing.T) {
+	t.Parallel()
+	runVaultEnterpriseClusterTest(t, "amazon-linux-ami", "ec2-user", os.Getenv("VAULT_AMI_TEMPLATE_VAR_DOWNLOAD_URL"))
```

Comment:
The use of an env var here won't be obvious to future maintainers. Consider add a method that:

1. Explicitly checks if the env var is set
1. Reports an error that says "Env var xxx not found. Please set it to a Vault Enterprise download URL to run this test."
1. Add a comment to the method explaining that Vault Enterprise URLs should be kept secret, so we use an env var to avoid putting it directly in our code or logs.